example/spec/parallel/pythonReferenceTests/PythonReferenceSpec.js

```diff
@@ -36,6 +37,10 @@ describe('The Python Reference workflow', () => {
     activityStep = new ActivityStep();
   });
 
+  afterAll(async () => {
+    await deleteExecution({ prefix: config.stackName, executionArn: workflowExecution.executionArn });
+  });
```

Comment:
Added this bit of cleanup. I was having integration test issues with my stack and noticed that this test wasn't cleaning up (verified that executions remain in Elastic after the test completes). This wasn't actually the cause of my problem but it seemed good to leave this in anyway.